bench/lib/context.go

```diff
@@ -54,7 +54,7 @@ type RuntimeContext struct {
 
 // NewRuntimeContext builds a runtime context from the config
 func NewRuntimeContext(cfg *Config) (*RuntimeContext, error) {
-	logger := cfg.Log.NewZapLogger()
+	logger, _ := cfg.Log.NewZapLogger()
```

Comment:
We should just panic here. having error in the result means the logger could be invalid.

Reply:
fixed